content/en/writing/foreign-income.md

```diff
@@ -0,0 +1,53 @@
+---
+contributors:
+- Nirant Kasliwal
+date: '2023-09-21 23:29:18+05:30'
+description: 'Quick Notes on Taxation of Foreign Income for Indian Residents'
+draft: false
+excerpt: ''
+featured_image: ''
+images: []
+lastmod: '2020-09-21 23:29:18+05:30'
+tags:
+- niranting
+title: Tax Tips for Consultants with Foreign Income
+toc: true
+weight: 50
+---
+
+Managing taxation and financial compliance as a consultant in India involves several considerations: GST to the benefits of Section 44ADA and the complexities of foreign entity considerations, consultants must juggle multiple aspects to ensure compliance and tax optimization. I'm sharing what I know in the hopes it's helpful to others.
+
+
+### 1. Goods and Services Tax (GST)
+
+- **Threshold**: Acquiring a GST number becomes almost mandatory once your income surpasses some X lakhs INR in a given financial year. This step legitimizes your income and ensures compliance with Indian taxation laws. The X limits keeps changing and I'd recommend you check the latest one with a CA and NOT rely on Google for this.
```

Comment:
> The X limits keeps changing and I'd recommend you check the latest one with a CA and NOT rely on Google for this.

Afaik, X = 20 lakhs for all the states except some northeastern ones (for them it's 10 Lakhs). Or is it more complicated than that? Even if it's more complicated, maybe you can still mention this and ask to cross check.

Reply:
You can also mention services like ClearTax for easy registration.